optaplanner-persistence/optaplanner-persistence-jackson/src/main/java/org/optaplanner/persistence/jackson/impl/domain/solution/JacksonSolutionFileIO.java

```diff
@@ -25,6 +25,7 @@ public JacksonSolutionFileIO(Class<Solution_> clazz) {
     public JacksonSolutionFileIO(Class<Solution_> clazz, ObjectMapper mapper) {
         this.clazz = clazz;
         this.mapper = mapper;
+        mapper.findAndRegisterModules();
```

Comment:
I'm going to submit a refactor to this later.
Jackson apparently does this via service loader now; we should use it too.